java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -121,4 +123,21 @@ public void lastsave() {
         var response = client.exec(new Transaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    @Test
+    @SneakyThrows
+    public void zrank_zrevrank_withscores() {
+        String zSetKey1 = "{key}:zsetKey1-" + UUID.randomUUID();
+        Transaction transaction = new Transaction();
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.2.0")) {
+            transaction.zadd(zSetKey1, Map.of("one", 1.0, "two", 2.0, "three", 3.0));
+            transaction.zrankWithScore(zSetKey1, "one");
+            transaction.zrevrankWithScore(zSetKey1, "one");
```

Comment:
nit
```suggestion
            Transaction transaction = new Transaction()
                    .zadd(zSetKey1, Map.of("one", 1.0, "two", 2.0, "three", 3.0))
                    .zrankWithScore(zSetKey1, "one")
                    .zrevrankWithScore(zSetKey1, "one");
```

Reply:
This seems to be the convention so far are we changing it?

Reply:
nope, both variants are good